app/3.0/docs/glossary.md

```diff
@@ -23,50 +20,28 @@ The [Polymer CLI](#polymer-cli) is designed to work with apps that use an app sh
 See [Instant Loading Web Apps with an Application Shell 
 Architecture](https://developers.google.com/web/updates/2015/11/app-shell) for more information.
 
-## B
-### Bower
-Bower is a package manager for web projects. Bower automates the installation of packages like 
-custom elements and JavaScript libraries. Polymer distributes its core library and elements via 
-Bower. See [bower.io](https://bower.io) for more information.
+### bundling
 
-### bower.json
-`bower.json` is a manifest used by Bower for configuring packages for use as dependencies. See [the 
-`bower.json` specification](https://github.com/bower/spec/blob/master/json.md) for more information.
+Bundling concatenates an app's scripts and resources to reduce the number of network requests it takes to load dependencies. 
 
-## C
-### crisper
-crisper extracts inline scripts from an HTML file. crisper is useful for separating JavaScript from 
-HTML in order to deploy in an environment that uses CSP (Content Security Policy).
+By contrast, if both server and browser support HTTP/2 (a newer protocol that permits concurrent resource requests), it can be optimal to leave resources unbundled. Unbundled builds permit lazy loading of dynamic dependencies. By default, Polymer apps are unbundled.
```

Comment:
It's never really optimal. Acceptable?

Reply:
SG

Reply:
I made the change to "acceptable". 

@justinfagnani @arthurevans For my own understanding, can you explain more about why it's never optimal to have unbundled resources? I thought bundling was suboptimal with lazy loading, and therefore unbundled builds would be optimal.